custom_model_runner/datarobot_drum/drum/drum.py

```diff
@@ -415,7 +432,8 @@ def _run_fit_and_predictions_pipelines_in_mlpiper(self):
             spark_jars=None,
         )
 
-        _pipeline_executor = Executor(config).standalone(True)
+        _pipeline_executor = Executor(config).standalone(True).set_verbose(self.options.verbose)
+        _pipeline_executor.set_logger(self.logger)
 
```

Comment:
what is this?

Reply:
I'm assigning current logger to mlpiper.Executor, to be able in mlpiper check logging level of drum logger.

Reply:
Can you leave a comment about that in the code? That's pretty confusing

Reply:
done